src/VehicleInterface.c

```diff
@@ -147,7 +147,7 @@ int vsc_read_next_msg(VscInterfaceType* vscInterface, VscMsgType *newMsg) {
 	VscMsgType *msgPtr;
 	int retval = -1;
 	bool done = false;
-	int bytesRead;
+	int bytesRead = 0;
```

Comment:
I like that explicit initialization.
Curious, Is that simply to match the intialization of the other variables or did you find that is caused something unexpected?

Reply:
I wanted to avoid an uninitialized value causing buffer overflow, in case the connection couldn't be established.